doc-content/enterprise-only/pages/building-custom-dashboard-widgets-running-heatmap-components-proc.adoc

```diff
@@ -0,0 +1,40 @@
+[id='building-custom-dashboard-widgets-running-heatmap-components-proc_{context}']
+
+= Running heatmap components
+
+You can run the internal components locally, outside of {CENTRAL} for testing purpose only. The same API is used to create external components. You can build your own components using an external components.
+For more information about external components, see <<building-custom-dashboard-widgets-external-components-con_{context}>>
+
+To run the specific heatmap component, complete the following steps:
+
+.Prerequisites
+
+* npm is installed on your system. For more information about installing npm, see https://docs.npmjs.com/downloading-and-installing-node-js-and-npm[Downloading and installing Node.js and npm].
+* Yarn is installed on your system. for more information about installing yarn, See https://classic.yarnpkg.com/en/docs/install/#debian-stable[2 - Installation].
+
+.Procedure
+
+. Go to `appformer/dashbuilder/dashbuilder-shared/dashbuilder-js` directory.
```

Comment:
This is not providing enough information. I would add into the "Prerequisites" section that to run the components outside of business central, the user needs to clone the [Appformer repository](https://github.com/kiegroup/appformer/) and after that, they can navigate in the command line into the directory that you are mentioning in this step

Reply:
Added this in the **Prerequisites** section.